webui/package.json

```diff
@@ -1,19 +1,65 @@
 {
-  "name": "root",
-  "private": true,
-  "dependencies": {
-    "lerna": "~3.13.0"
-  },
-  "engines": {
-    "node": "10.11.0",
-    "npm": "6.4.1"
-  },
-  "scripts": {
-    "bootstrap": "lerna bootstrap",
-    "start": "lerna run start --parallel --stream",
-    "build": "lerna run build --parallel --stream",
-    "test-ci": "lerna run test-ci --parallel --stream",
-    "coverage-ci": "lerna run coverage-ci --parallel --stream",
-    "format": "lerna run format --parallel --stream"
-  }
+	"name": "root",
```

Comment:
is the update in this file necessary?

Reply:
Sorry, this was introduced by web-based auto-merge.
It is not needed.
Let me remove it.